pantos-io/servicenode
# [PAN-2158] Integrate API docs

## What type of PR is this? (check all applicable)


## Description

Code required to build API docs using Swagger.

## Related Tickets & Documents



- Related Issue #
- Closes #

## QA Instructions

Check the output from the corresponding CI entry is correct. I had to create an additional `create_application` entry to avoid all the database and celery initialization that are not required for the docs generation.


## [optional] Are there any post deployment tasks we need to perform?
